lib/Backends/CPU/tests/CPUOperatorTest.cpp

```diff
@@ -0,0 +1,146 @@
+#include "tests/unittests/OperatorTest.h"
+
+#include <iostream>
+
+using namespace glow;
+
+IS_TEST_ENABLED(std::string name) {
+  std::set<std::string> blacklist = {
+      "less_float16Cases/0",
```

Comment:
`/0` is correct here.  Since we're instantiating each test suite with a single backend, they're all suffixed with `/0`.  This is different from the current setup, which instantiates the suite with a set of values.